### PR TITLE
feat: Snapshot widget

### LINKED ISSUE #976: [Governance Visibility] Snapshot safe-app widget
## What is the feature about

Create a new Safe App that tracks Safe DAO snapshot proposals.

## The list of requirements
- Navigate to external Snapshot proposal URL
- Navigate to external Snapshot space
- ~Cast a vote~
- Navigate to Safe Forum

See proposal details:
  - End date
  - Percentage of quorum reached
  - Current results

## Designs/sketches
https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=34%3A3627

## Links
Epic: #969 
Safe's Snapshot space: https://snapshot.org/#/safe.eth

### PR DESCRIPTION
**Pull request by iamacook**
## What it solves

Resolves #976

## How this PR fixes it

A basic Snapshot widget which lists the SafeDAO proposals has been created, matching the [proposed design](https://www.figma.com/file/8nKOe7h90OS7jmn3nksbq9/Governance-visibility-widgets?node-id=34%3A2394).

## How to test it

Open the dashboard and observe the new widget which matches the above design:

- Clicking a proposal should open it in a new window, as well as the "View all" and "SafeDAO Forum" links.
- Reducing the resolution should truncate the title correctly, leaving the other columns visible.

## Screenshots

widget

image

image

image

### CODE REVIEW

File: src/config/routes.ts
Comment (iamacook) [inline src/config/routes.ts:8]: I added this for testing purposes. I will remove it, as well as `demo.tsx`, before merging.

File: src/components/dashboard/GovernanceSection/GovernanceSection.tsx
Comment (DiogoSoaress) [inline src/components/dashboard/GovernanceSection/GovernanceSection.tsx:15]: The state and the onClick handler are already handled by the `<Accordion />`

File: src/components/dashboard/SnapshotWidget/index.tsx
Comment (DiogoSoaress) [inline src/components/dashboard/SnapshotWidget/index.tsx:10]: do we need a constant for this? only used in the `Chip`

File: src/components/dashboard/SnapshotWidget/index.tsx
Comment (DiogoSoaress) [inline src/components/dashboard/SnapshotWidget/index.tsx:65]: We should have a different constant name like "DISPLAYED_PROPOSALS" and go with 3 as in the mockups.

**Review by DiogoSoaress — DISMISSED** [review]
While this PR looks great and follows the designs perfectly, should we not hold it on in this repo preferring the implementation of the widget in the safe apps? I would wait for https://github.com/safe-global/web-core/pull/1149 and https://github.com/safe-global/safe-react-apps/pull/569 to be merged first. WDYT?

File: src/components/dashboard/GovernanceSection/GovernanceSection.tsx
Comment (iamacook) [reply src/components/dashboard/GovernanceSection/GovernanceSection.tsx:15]: I've removed the state.

File: src/components/dashboard/SnapshotWidget/index.tsx
Comment (iamacook) [reply src/components/dashboard/SnapshotWidget/index.tsx]: I've moved it to the relevant property/CSS module.

File: src/components/dashboard/SnapshotWidget/index.tsx
Comment (iamacook) [reply src/components/dashboard/SnapshotWidget/index.tsx]: I've adjusted it to `3` and now pass it to `useSnapshot`.

**Review by DiogoSoaress — COMMENTED** [review]
Code looks good to me know 👍 . But let's keep it as not merge until we figure out where this widget should live.

**Comment by DiogoSoaress** [comment]
Closing in favor of https://github.com/safe-global/safe-react-apps/pull/575

**Pull request closed without merging**
